archive/imo/imo2021_q1.lean

```diff
@@ -0,0 +1,264 @@
+import tactic data.real.sqrt
+
+noncomputable theory
+
+open real
```

Comment:
you don't need the `noncomputable theory`, and furthermore the only thing allowed module docstrings is the `import`s